sdks/nodejs/spec/agonesSDK.spec.js

```diff
@@ -0,0 +1,246 @@
+const EventEmitter = require('events');

```

Comment:
We need to run `npm test` while inside the `sdks/nodejs` folder. Should we copy the folder in to the image to achieve this?

Reply:
I think this is the last item. Since we already have node installed on the build image, this should hopefully be pretty straight forward.

We should have a `test-node` target in the Makefile.  Probably something like:

```
test-node: $(ensure-build-image)
	docker run --rm $(common_mounts) -w $(mount_path)/sdk/nodejs $(build_tag) npm list test
```
(not sure if I got the commands 100% right, but let me know if you get my gist?)

Then also add this target to `test`, and I think we are good to go.